apis/apps/v1alpha1/nebulacluster_types.go

```diff
@@ -66,6 +66,10 @@ type NebulaClusterSpec struct {
 	// +optional
 	SchedulerName string `json:"schedulerName"`
 
+	// Flag to enable/disable pvc reclaim while the nebula cluster deleted , default false
+	// +optional
+	EnablePVReclaim *bool `json:"enablePVReclaim,omitempty"`
```

Comment:
why don't use `EnablePVCReclaim` as parameter name

Reply:
Reclaim behavior is used for pv